ADDED FLY FIGHTER GAME

TheNikhilRaj opened the pull request:
## PR Description 📜
I created a simple game Fly Fighter building with JS.

Description
Overview : This is a very interesting and innovative , the users are gonna love this game.

Fixes # #4771 

<hr>
 
## Mark the task you have completed ✅

<!----Please delete options that are not relevant. In order to tick the check box just but x inside them for example [x] like this----->

- [ ] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ ] I have performed a self-review of my own code or work.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes generates no new warnings.
- [ ] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [ ] I have added screenshot for website preview in assets/images 
- [ ] I have added entries for my game in main README.md
- [ ] I have added README.md in my folder 
- [ ] I have added working video of the game in README.md (optional)
- [ ] I have specified the respective issue number for which I have requested the new game.


<hr>

## Add your screenshots(Optional) 📸

![346280545-e9529581-9d3e-4494-8eea-1b0a342de5cb](https://github.com/kunjgit/GameZone/assets/129590267/41f13e8f-9b67-43f1-b014-1f6a4ab84188)



--- 
<br>
@kunjgit  task completed!

## Thank you soo much for contributing to our repository 💗

ishita-43 reviewed (changes requested):
Hi there! you need to make following  changes 
Add your game entry in the main Readme.md file of the gamezone refer other entries there for the format 
Add your game image in the asset folder of the gamezone follow contribution guidelines while naming the image

TheNikhilRaj commented:
@ishita-43 as you had said i updated the following changes.

ishita-43 reviewed (changes requested):
Please make the following changes 
add your game entry in main readme.md file of the repo by editing it 
add your game  image in the asset folder of the gamezone follow naming convention while naming the image

ishita-43 commented:
> @kunjgit @ishita-43 as you had said i updated the following changes.

you will find the readme file in the gamezone repo along with asset folder and games folder

TheNikhilRaj commented:
> Please make the following changes 
> add your game entry in main readme.md file of the repo by editing it 
> add your game  image in the asset folder of the gamezone follow naming convention while naming the image 

@ishita-43 But I have done these things, please you can check it once.

TheNikhilRaj commented:
> > @kunjgit @ishita-43 as you had said i updated the following changes.
> 
> you will find the readme file in the gamezone repo along with asset folder and games folder  

It's already done. @ishita-43

ishita-43 commented:
> > > @kunjgit @ishita-43 as you had said i updated the following changes.
> > 
> > 
> > you will find the readme file in the gamezone repo along with asset folder and games folder
> 
> It's already done. @ishita-43

you are saying that your  game exists in below file https://github.com/kunjgit/GameZone/blob/main/README.md

TheNikhilRaj commented:
> > > > @kunjgit @ishita-43 as you had said i updated the following changes.
> > > 
> > > 
> > > you will find the readme file in the gamezone repo along with asset folder and games folder
> > 
> > 
> > It's already done. @ishita-43
> 
> you are saying that your game exists in below file https://github.com/kunjgit/GameZone/blob/main/README.md

@ishita-43 yes i have added my game many times in this file https://github.com/kunjgit/GameZone/blob/main/README.md

ishita-43 commented:
The thing is you need to add your game in main readme file and make changes in this pr only as only one pr is allowed is allowed in this repository so make sure that you make your changes in this pr only by updating your pr
